Help/FAQ.rst

```diff
@@ -138,3 +138,83 @@ our master branch. Right now we use 7 of those checks:
 
 - **continuous-integration/appveyor/pr** This one does the same as the one
   above, but for Windows, both 32 and 64bits versions.
+
+What is coala Community Code of Conduct?
+----------------------------------------
+
+Like the technical community as a whole, the coala team and community is
```

Comment:
Please dont include the entire CoC.

Only an item about the CoC, so people know we have one, and how to find it.